tests/postgres_tests/test_serialfield.py

```diff
@@ -0,0 +1,64 @@
+from tests.postgres_tests import PostgreSQLTestCase
+from tests.postgres_tests.models import SerialModel
+
+from django.contrib.postgres.fields.serial import BigSerialField, SerialField
+from django.db import models
+from django.db.backends.postgresql.base import \
+    DatabaseWrapper as PostgresDatabaseWrapper
+from django.test import SimpleTestCase, skipUnlessDBFeature
+from django.test.utils import isolate_apps
+
+
+@isolate_apps('invalid_models_tests')
+class SerialFieldModelTests(SimpleTestCase):
+    def test_not_unique(self):
+        class Model(models.Model):
+            id = SerialField(unique=True)
+
+        field = Model._meta.get_field('id')
+        errors = field.check()
+        expected = []
+        self.assertEqual(errors, expected)
+
+    def test_primary_key(self):
+        # primary_key must be True. Refs #12467.
```

Comment:
I don't see the relation with #12467 (More helpful error message when loading fixture with invalid date)?